plugins/rss/komand_rss/connection/connection.py

```diff
@@ -1,13 +1,24 @@
-import komand
-from .schema import ConnectionSchema
+import insightconnect_plugin_runtime
+from .schema import ConnectionSchema, Input
 
 # Custom imports below
+from insightconnect_plugin_runtime.exceptions import ConnectionTestException
+from typing import Dict
+import validators
 
 
-class Connection(komand.Connection):
+class Connection(insightconnect_plugin_runtime.Connection):
     def __init__(self):
         super(self.__class__, self).__init__(input=ConnectionSchema())
-        self.FEED_URL = None
+        self.feed_url = None
 
-    def connect(self, params):
-        self.FEED_URL = params.get("url")
+    def connect(self, params={}) -> None:
+        self.feed_url = params.get(Input.URL, "")
+
+    def test(self) -> Dict[str, bool]:
+        if not validators.url(self.feed_url):
+            raise ConnectionTestException(
```

Comment:
Are there any basic api calls we can make with this service to test for 200 ?

Reply:
Yup, I can try to fetch entries there. Good catch. I'll extend it 😄